Source/WebCore/loader/cache/CachedResourceLoader.cpp

```diff
@@ -1169,10 +1168,9 @@ ResourceErrorOr<CachedResourceHandle<CachedResource>> CachedResourceLoader::requ
                 if (auto metricsFromResource = resource->takeNetworkLoadMetrics(); metricsFromResource && documentLoader && metricsFromResource->redirectStart >= documentLoader->timing().timeOrigin())
                     metrics = WTFMove(metricsFromResource);
                 auto resourceTiming = ResourceTiming::fromMemoryCache(url, request.initiatorType(), loadTiming, resource->response(), metrics ? *metrics : NetworkLoadMetrics::emptyMetrics(), *request.origin());
-                if (initiatorContext == InitiatorContext::Worker) {
-                    ASSERT(is<CachedRawResource>(resource.get()));
+                if (initiatorContext == InitiatorContext::Worker)
                     downcast<CachedRawResource>(resource.get())->finishedTimingForWorkerLoad(WTFMove(resourceTiming));
```

Comment:
I guess we feel confident this is always a `CachedRawResource`, since we would have seen crashes here previously. I'm sort of wondering if doing a `dynamicDowncast` with nullptr check would be a "better" (or at least less-likely-to-crash) approach.